src/transport/msg_providers/kafka.py

```diff
@@ -46,19 +46,32 @@ def __init__(self, message_schema: Schema, client: str):
         logger.info({'msg': f'Subscribe to "{kafka_topic}".'})
         self.kafka.subscribe([kafka_topic])
 
-        super().__init__(client, message_schema)
+        super().__init__(message_schema)
 
     def __del__(self):
         self.kafka.close()
 
-    def _receive_message(self) -> Optional[dict]:
+    def _receive_message(self) -> Optional[str]:
```

Comment:
Why now this is str?
```suggestion
    def _receive_message(self) -> str | None:
```

Reply:
Done